### PR TITLE
[PLAY-718] Kits in HTML and CSS

### PR DESCRIPTION
**Pull request by nickamantia**
**What does this PR do?** 
[Runway](https://nitro.powerhrg.com/runway/backlog_items/PLAY-718)

This PR adds a `Copy HTML` button to rails doc examples that copies the example's HTML.

**Screenshots:** Screenshots to visualize your addition/change

Screen Shot 2023-04-24 at 2 53 53 PM

**How to test?** Steps to confirm the desired behavior:
1. Go to any Rails doc
2. Click on the Copy HTML button
3. Paste somewhere to see what was copied
4. Do the same on a different kit and see if the copied code is different

#### Checklist:
- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_docs/copyCode.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const copyContent = (content) => {
+  const copyText = document.createElement("textarea")
+  copyText.value = content
+  document.body.appendChild(copyText)
+  copyText.select()
+  document.execCommand("copy")
```
Comment (augustomallmann) [inline playbook/app/pb_kits/playbook/pb_docs/copyCode.js:6]: the `document.execCommand` method [has been deprecated ](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand).

What do you think of replacing this method with the `writeText` of the [clipboard api?](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) 

```suggestion
  navigator.clipboard.writeText(content);
```

Comment (augustomallmann) [inline playbook/app/pb_kits/playbook/pb_docs/copyCode.js:6]: the `document.execCommand` method [has been deprecated ](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand).

What do you think of replacing this method with the `writeText` of the [clipboard api?](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) 

```suggestion
  navigator.clipboard.writeText(content);
```

**Review by augustomallmann — COMMENTED** [review]
Great job, Nick! Left you a small suggestion

File: playbook/app/pb_kits/playbook/pb_docs/copyCode.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickamantia) [reply playbook/app/pb_kits/playbook/pb_docs/copyCode.js:6]: Nice catch!! i will switch this out, thanks!

[the remark this answers is not in the document]
Comment (nickamantia) [reply playbook/app/pb_kits/playbook/pb_docs/copyCode.js]: Push this change up, thanks again for the suggestion @augustomallmann!!!

File: playbook-website/Gemfile.lock [hunk 2]
```diff
@@ -285,6 +285,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  x86_64-darwin-20
```
Comment (nidaqg) [inline playbook-website/Gemfile.lock:288]: @jasperfurniss this is definitely in Nitro as well, so we should probably be fine shipping it, will leave it to you to make a final call on this. Rest of the code is good to go!

**Review by nidaqg — APPROVED** [review]

File: playbook/app/pb_kits/playbook/playbook-rails.js [hunk 3]
```diff
@@ -9,6 +9,10 @@ window.datePickerHelper = datePickerHelper
 import dialogHelper from './pb_dialog/dialogHelper'
 window.dialogHelper = dialogHelper
 
+// CopyContent
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/playbook-rails.js:12]: @jasperfurniss I'm not sure we want this here. This file is intended for kits not docs.

**Review by jasoncypret — CHANGES_REQUESTED** [review]

File: playbook/app/pb_kits/playbook/playbook-rails.js (hunk 3 above)
Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/playbook-rails.js:12]: I'll go ahead and move this elsewhere.

**Review by jasperfurniss — APPROVED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**